src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -516,6 +516,86 @@ default Path resolve(String other) {
         return resolve(getFileSystem().getPath(other));
     }
 
+    /**
+     * Resolves a path or a sequence of paths against this path.
```

Comment:
"or"  -> "and then"

Reply:
Like in `Path.of`, the first parameter may be considered element zero of the sequence which was the intent here.